cmd/trigger_ui/main.go

```diff
@@ -207,18 +207,22 @@ func (s *TriggerServer) vxExportPOST(ctx *gin.Context) {
 	}
 
 	go func() {
-		err := s.vidispine.SetItemMetadataField(vxID, "", vscommon.FieldExportAudioSource.Value, audioSource)
+		err := s.vidispine.SetItemMetadataField(vsapi.SetItemMetadataFieldParams{
+			ItemID: vxID,
+			Key:    vscommon.FieldExportAudioSource.Value,
+			Value:  audioSource,
+		})
 		if err != nil {
 			log.Default().Println(err)
 		}
 
 		for i, element := range languages {
-			if i == 0 {
-				err = s.vidispine.SetItemMetadataField(vxID, "", vscommon.FieldLangsToExport.Value, element)
-			} else {
-				err = s.vidispine.AddToItemMetadataField(vxID, "", vscommon.FieldLangsToExport.Value, element)
-
-			}
+			err = s.vidispine.SetItemMetadataField(vsapi.SetItemMetadataFieldParams{
+				ItemID: vxID,
+				Key:    vscommon.FieldLangsToExport.Value,
+				Value:  element,
+				Add:    i != 0,
```

Comment:
This was intentionally left as two functions. It's the implicit vs. explicit thing.
The behavior is significantly different that IMO it should not be controlled by just an optional parameter.

Even worse, the implicit default is false, i.e. "replace".
You are one forgotten `Add: true` away from nuking a metadata field in an automated fashion.

Reply:
split it back up, so this PR is basically just about the params and cleanup now